kuka_quantec_support/config/kr210_r2700_2_joint_limits.yaml

```diff
@@ -11,31 +11,37 @@ default_acceleration_scaling_factor: 1.0
 joint_limits:
   joint_1:
     has_velocity_limits: true
-    max_velocity: 2.0944
```

Comment:
let's find the right values, if the old ones are not correct, please also modify in the urdf

Reply:
joint 2 also differs, please also modify that